extension/li3ds--1.0.0.sql

```diff
@@ -72,7 +78,11 @@ create table session(
 
 create table datasource(
     id serial primary key
-    , uri varchar
+    -- possible uri schemes are "file" and "column"
+    , uri varchar not null constraint uri_scheme check (split_part(uri, ':', 1) = any ('{file, column}'::text[]))
+    , type datasource_type not null
+    , parameters jsonb
+    , capture_time timestamptz
```

Comment:
This should be a time interval to accomodate for non-instantanneous datasources, (eg : lidar, route).
I would directly go to a 3D+T bbox, expressed in the datasource referential.